docs/run-keep-ecdsa.adoc

```diff
@@ -20,22 +20,27 @@ these capabilities consider the following criteria:
   application port running under the same IP.
 - Recommended machine types by provider:
 
+Your operating environment will ultimately dictate what machine type to go with.  This is
+particulary relevant if you're running a containerized solution where multiple applications are
+sharing VM resources.  The below types are sufficient for running at least one instance of the
+Keep ECDSA client.
+
 [%header,cols=2*]
 |===
 |Cloud Provider
 |Machine Type
 
 |Google Cloud
-|n1-standard-4
+|n1-standard-2
 
 |AWS
-|m5.xlarge
+|t3.medium
 
 |Azure
-|D4s v3
+|A2 v2
```

Comment:
For our keep network use, `F2s v2` may be a good fit. F series is compute-optimized, hence could handle messages processing peaks and prime numbers generation better. F is slightly cheaper than A and I believe it has broader availability. A series is recommended for dev/test usage.

Reply:
Having a look.

Reply:
Yeah this looks appropriate, making the update - also double  checking the `c` class on AWS.

Reply:
Updated both  - prices were close to what was set.   There's some other options for GCP  but need to  make sure 2gb is safe.